app/views/layouts/_js_exception_handler.erb

```diff
@@ -3,7 +3,32 @@
   <script>
   var _rollbarConfig = {
       accessToken: "<%= ENV['JS_EXCEPTION_LOGGER_KEY'] %>",
+      // This should suppress errors from third-party scripts:
+      // https://docs.rollbar.com/docs/reduce-noisy-javascript-errors#section-whitelist-specific-domains
+      hostWhiteList: ['lib.mit.edu', 'herokuapp.com'],
       captureUncaught: true,
+      checkIgnore: function(isUncaught, args, payload) {
+        // This suppresses an iOS specific error not relevant to our code.
+        if (window.__gCrWeb) {
```

Comment:
I'm not sure if these rules should be put in this config or just ignored permanently in the rollbar UI. Did you consider both and this is better?